cookbooks/van-keken-vof.prm

```diff
@@ -0,0 +1,118 @@
+# A description of the van Keken et al. benchmark. See the manual for more
+# information.
+
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 2000
+set Use years in output instead of seconds = false
+set CFL number                             = 0.5
+set Output directory                       = output-van-keken-vof-h
+
+subsection Geometry model
+  set Model name = box
+  subsection Box
+    set X extent  = 0.9142
+    set Y extent  = 1.0000
+  end
+end
+
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = left, right
+  set Zero velocity boundary indicators       = bottom, top
+end
+
+
+subsection Material model
+  set Model name = simple
+  subsection Simple model
+    set Reference density             = 1010
+    set Viscosity                     = 1e2
+    set Thermal expansion coefficient = 0
+  end
+end
+
+subsection Gravity model
+  set Model name = vertical
+  subsection Vertical
+    set Magnitude = 10
+  end
+end
+
+
+############### Parameters describing the temperature field
+# Note: The temperature plays no role in this model
+
+subsection Boundary temperature model
+  set List of model names = box
+end
+
+subsection Initial temperature model
+  set Model name = function
+  subsection Function
+    set Function expression = 0
+  end
+end
+
+
+############### Parameters describing the compositional field
+# Note: The compositional field is what drives the flow
+# in this example
+
+subsection Compositional fields
```

Comment:
There are also some modifications to the CFL number, specifying the use of a discontinuous composition discretization, and the postprocessing step.

I did not model it on that approach due in part to the fact that the smooth vs discontinuous prm file did not use that approach either.

Reply:
@class4kayaker - Given that the parameter still share many common features, I would still derive the current parameter VOF from one of the two original files. If updates are made to the benchmark later on, this ensures the current file also gets updated as well.

Reply:
Doing a grep on all of the cookbook parameter files, I do not see any other cookbook that makes use of the approach you suggest. Additionally, it would create a dependency on the referenced file remaining in the same relative location if the cookbook is copied as a reference. As such, I am not sure that including the file is the appropriate way to set up the cookbook.